packages/rrweb-snapshot/src/rebuild.ts

```diff
@@ -77,35 +79,61 @@ export function addHoverClass(cssText: string, cache: BuildCache): string {
   }
 
   const selectors: string[] = [];
-  ast.stylesheet.rules.forEach((rule) => {
+  const medias: string[] = [];
+  function getSelectors(rule: any) {
```

Comment:
There is some types you could use from `./css.ts` to type this.
```suggestion
  function getSelectors(rule: StyleRules) {
```

Reply:
You might need to change the name of `rule` here though --^

Reply:
This is actually using css.ts which has it's own interface.
I wasted a half hour trying to figure out the typings.  I can't see the benefit of typings, it has to iterate over note only StyleRules types but implementations of Node interface (see css.ts)

I get the following error if I add `Node` there:

```
src/rebuild.ts:85:30 - error TS2339: Property 'forEach' does not exist on type '{}'.

85       (rule.selectors || []).forEach((selector: string) => {
```

Reply:
Maybe I can hop on a call with you, we can try and figure it out together?